### ISSUE TITLE
Breadcrumbs are not Identified as Breadcrumbs for Screen Reader Users

### ISSUE THREAD

**Issue by mtribone** [samvera/hyrax#3068]
### Descriptive summary

In Hyrax Version: 2.1.0.rc2, the breadcrumbs are not identified as breadcrumbs for screen reader users. Needs to be handled system-wide for public facing and internal admin tasks like the dashboard.

### Rationale

Additional measures need to be taken to ensure that the controls provide important information to assistive technologies and allow themselves to be controlled by assistive technologies. The breadcrumbs are more than just a list of links and provide context within a system.

### Expected behavior

Expect to find `role="region"` and `aria-label="breadcrumbs"` to the element that contains the breadcrumbs. 

### Actual behavior

No additional information outlining that these links are indeed breadcrumbs.

### Steps to reproduce the behavior
Public Collections
1. Navigate to Nurax Home and search for publications 
1. Select the UC San Diego History user collection
1. Select the [Charles H. Graves Photographs and Films](https://nurax.curationexperts.com/collections/xw42n795p?locale=en) subcollection
1. Right click the breadcrumbs at the top of the content area and inspect the HTML

Dashboard
1. Navigate to [Nurax Dashboard](https://nurax.curationexperts.com/dashboard?locale=en)
1. Select Works from the left-hand menu under Repository Contents
1. Add New Work
1. Choose Image or Generic Work
1. Right click the breadcrumbs at the top of the content area and inspect the HTML

### Related work

Violates [WCAG 4.1.2](https://www.w3.org/TR/UNDERSTANDING-WCAG20/ensure-compat-rsv.html) guidelines

**Comment by mtribone** [comment]
![screen shot 2018-05-14 at 2 56 50 pm](https://user-images.githubusercontent.com/4163828/40017348-23350052-5787-11e8-90b1-4ce6d547ead4.png)
